### ISSUE TITLE
Redundant display on finish!()

### ISSUE THREAD

**Issue by tomyun** [timholy/ProgressMeter.jl#199]
I believe this started happening since the recent update to ProgressMeter 1.6.0.

<img width="702" alt="Screen Shot 2021-05-05 at 12 38 10 PM" src="https://user-images.githubusercontent.com/906626/117199199-c23abf00-ad9e-11eb-9082-453cd962a506.png">

```
julia> versioninfo()
Julia Version 1.6.1
Commit 6aaedecc44 (2021-04-23 05:59 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.7.0)
  CPU: Intel(R) Core(TM) i7-10700K CPU @ 3.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, skylake)
```

IJulia is 1.23.2.

**Comment by tomyun** [comment]
Indeed, it happens on REPL too.

```
julia> p = Progress(10; dt=0.1)
       for i in 1:10
           sleep(0.1)
           ProgressMeter.next!(p)
       end
       ProgressMeter.finish!(p)
Progress: 100%|████████████████████████████████████| Time: 0:00:01
Progress: 100%|████████████████████████████████████| Time: 0:00:01
```

**Comment by IanButterworth** [comment]
Thanks for the report. Some logic was omitted in #198 that made `finish!` print each time.

Fixed by #200
